pep-9999.rst

```diff
@@ -0,0 +1,284 @@
+
+    ==================  ===========  ===============================================
+    Attribute           Type         Description
+    ==================  ===========  ===============================================
+    cert                str          Path to certificate authority bundle (*.pem)
+    cert-client         str          Path to client certificate (*.pem)
+    enabled             bool         Whether the repository should be used or not (default: true)
+    package_index_name  str          The corresponding package index name, defined in ``config.toml``
+    password            str          The password for the repository
+    proxy_password      str          The password for the proxy server
+    proxy_username      str          The username for the proxy server
+    username            str          The username for the repository
+    ==================  ===========  ===============================================
+
+
+Tools preferring to look up credentials from other locations, such as keyring, are
+advised to use the full package index URL as the key, rather than the index name or
+hostname.
+
+
+The auth.toml file
+''''''''''''''''''
+
+Optionally, authentication can be defined in a separate ``auth.toml``. This way,
+credentials can be fully or partially separated from package index URLs.
+
+Example ``auth.toml``::
+
+    [[credentials]]
+    package_index_name = "myrepo1"
+    username = "foo"
+    password = "YmFyCg=="
+
+
+Package discovery
+-----------------
+
+PyPI.org
+''''''''
+
+The PyPI.org repository (``repositories.pypi``) does not have to be specified in the
+``config.toml``. If not specified, it will be used as the first package index by
+default. However, by explicitly defining it, the repository attributes can be used and
+it can be prioritized differently.
+
+
+Package index priority
+''''''''''''''''''''''
+
+Tools should respect priority by acting as if all lower priority package index were not
+specified when a higher priority package index satisfies the task. Tools that choose not
+to respect priority MUST display a warning to users when a non-zero priority value is
+being ignored. It is acceptable for tools to be less efficient in order to handle
+prioritized package indexes, for example, by not performing queries in parallel.
+
+
+Configuration file discovery
+----------------------------
+
+The locations of the configuration files (``config.toml`` and ``auth.toml``) vary slightly across platforms. You may have global, per-user and per-environment configuration:
+
+If multiple configuration files are found, then they are merged in the following order:
+
+1. The global files are read
+2. The per-user files are read
+3. The per-environment files are read
+
+
+Global
+''''''
+
+* Unix: ``/etc/.config/python/``
+* macOS: ``/Library/Application Support/python/``
+* Windows: ``%PROGRAMDATA%\Python``
+
+
+Per-user
+''''''''
+
+* Unix: ``$HOME/.config/python/``
```

Comment:
I would use `$XDG_CONFIG_HOME/python`, with a fallback default of `$HOME/.config/python`

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

Reply:
Yes, very nice. I added that in. Please have a look at resolve this conversation if you think it looks good.